response_writer.go

```diff
@@ -425,3 +426,30 @@ func (w *responseWriter) Push(target string, opts *http.PushOptions) error {
 		return err
 	}
 }
+
+type extWriter struct {
+	rw *responseWriter
+}
+
+func (w *extWriter) Write(p []byte) (n int, err error) {
+	return w.rw.Write(p)
+}
+
+func (w *extWriter) Flush() error {
+	w.rw.Flush()
```

Comment:
Why mute the raw flush error here?

Reply:
the raw flush function not has a error

Reply:
Just return `w.rw.Flush()` in case any possible change in future?

Reply:
`w.rw.Flush()` has no return value, if just return `w.rw.Flush()`, The compiler will complain：
> w.rw.Flush() (no value) used as value

Reply:
I see.. Strange design - so confident that is would be no error in `Flush`...